### DISCUSSION TITLE
Outline view in sidebar

### DISCUSSION THREAD

**Discussion by Arbiter313** [notaapp/talk#259] [Feature Requests] [answered]
It would be helpful to view an outline of your document (the structure of your document at the heading level) in a sidebar as is seen in Typora, Obsidian and Ulysses. It would also be useful to be able to go to that heading in the editor by clicking on the relevant heading in the outline, and even to be able to fold a heading from the outline. 

Thank you.

**Comment by astoilkov** [comment]
We've already implemented this internally. There is a high chance that the next version will include it!

### 🚀

**Reply by Arbiter313** [reply]
That sounds great, can't wait to use this feature!

**Reply by scottrmead** [reply]
Agreed!  I love using those editors, and Caret because it does such a great job of high-leveling the doc for you.  I'm loving nota and will probably move everything to it once I get outline and live preview.

**Comment by astoilkov** [comment, marked as answer]
We introduced a right sidebar in 0.38. I hope you like it. You can hide everything else except the outline if you want only an outline.
